core/templates/pages/preferences-page/form-fields/preferred-languages.component.ts

```diff
@@ -51,16 +65,18 @@ export class PreferredLanguagesComponent {
       }
     });
   }
-
+  
+  
```

Comment:
Please try to remove unnecessary changes from your PR. Look at the "Files Changed" tab on GitHub and make sure that it contains only the necessary changes, and no more.